tasks/task02/triangle_area.py

```diff
@@ -0,0 +1,45 @@
+import math
+
+def area_base_height():
+    values = input("Enter base and height: ").split(" ")
+    if len(values) != 2 or not all(i.isdigit() for i in values) or not all(int(i) >= 0 for i in values):
+        print(f"\nCheck values (only two positive numbers greater than zero are allowed).\n")
+        return area_base_height()
+    else:
+        base = int(values[0])
+        height = int(values[1])
+        return (height * base) / 2
+
+def area_2sides_angle():
+    values = input("Enter 2 sides and angle(degrees) between them: ").split(" ")
+    if len(values) != 3 or not all(i.isdigit() for i in values) or not all(int(i) >= 0 for i in values):
+        print(f"\nCheck values (only three positive numbers greater than zero are allowed).\n")
```

Comment:
`f` there is no need 

https://docs.python.org/3/tutorial/inputoutput.html#fancier-output-formatting